storage/gcloud/aio/storage/storage.py

```diff
@@ -57,6 +57,59 @@ def __init__(self, *, service_file: Optional[Union[str, io.IOBase]] = None,
     def get_bucket(self, bucket_name: str) -> Bucket:
         return Bucket(self, bucket_name)
 
+    async def copy(self, bucket: str, object_name: str,
```

Comment:
I like Allan's suggestion about renaming `bucket` to `source_bucket`, I think that would make this slightly clearer, as well as the code below where it is used.

Reply:
I think keeping with the style is important as well, and we use `bucket` throughout this file. If we really want the variable names to pair, maybe change `destination_bucket` instead? I suppose it could just be `destination`.